meta-xt-domd-gen4/recipes-bsp/arm-trusted-firmware/arm-trusted-firmware_git.bbappend

```diff
@@ -0,0 +1,7 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
In the commit message s/translation regime/translation mode

Reply:
This term is used by ARM ARM:

> In a translation regime that provides only one stage of address translation, the IPA is identical to the PA. Alternatively, the translation regime can be considered as having no concept of IPAs.

For example